promql/parser/parse.go

```diff
@@ -761,7 +761,7 @@ func (p *parser) checkAST(node Node) (typ ValueType) {
 		for _, l1 := range n.VectorMatching.MatchingLabels {
 			for _, l2 := range n.VectorMatching.Include {
 				if l1 == l2 && n.VectorMatching.On {
-					p.addParseErrf(opRange(), "label %q must not occur in ON and GROUP clause at once", l1)
+					p.addParseErrf(opRange(), "label %q must not occur in ON and GROUP clause more than once", l1)
```

Comment:
not sure on exact wording here, but the `at once` language confused me .. maybe `at the same time` is best?